binaries/speed_benchmark_torch.cc

```diff
@@ -82,10 +82,15 @@ int main(int argc, char** argv) {
     }
   }
 
+  auto qengines = at::globalContext().supportedQEngines();
```

Comment:
Shall we add same logic to android wrapper code? @IvanKobzarev

Reply:
Yes, I will create PR for it.

Reply:
Might need this for iOS too cc @xta0